public/themes/pterodactyl/js/admin/statistics.js

```diff
@@ -0,0 +1,121 @@
+var freeDisk = Pterodactyl.totalNodeDisk - Pterodactyl.totalServerDisk;
+let diskChart = new Chart($('#disk_chart'), {
+    type: 'pie',
+    data: {
+        labels: ['Free Disk', 'Used Disk'],
+        datasets: [
+            {
+                label: 'Disk in MBs',
+                backgroundColor: ['#51B060', '#ff0000'],
+                data: [freeDisk, Pterodactyl.totalServerDisk]
+            }
+        ]
+    }
+});
+
+var freeRam = Pterodactyl.totalNodeRam - Pterodactyl.totalServerRam;
+let ramChart = new Chart($('#ram_chart'), {
+    type: 'pie',
+    data: {
+        labels: ['Free RAM', 'Used RAM'],
+        datasets: [
+            {
+                label: 'RAM in MBs',
+                backgroundColor: ['#51B060', '#ff0000'],
+                data: [freeRam, Pterodactyl.totalServerRam]
+            }
+        ]
+    }
+});
+
+var activeServers = Pterodactyl.serverCount - Pterodactyl.suspendedServers;
+let serversChart = new Chart($('#servers_chart'), {
+    type: 'pie',
+    data: {
+        labels: ['Active', 'Suspended'],
+        datasets: [
+            {
+                label: 'Servers',
+                backgroundColor: ['#51B060', '#E08E0B'],
+                data: [activeServers, Pterodactyl.suspendedServers]
+            }
+        ]
+    }
+});
+
+let statusChart = new Chart($('#status_chart'), {
+    type: 'pie',
+    data: {
+        labels: ['Online', 'Offline', 'Installing', 'Error'],
+        datasets: [
+            {
+                label: '',
+                backgroundColor: ['#51B060', '#b7b7b7', '#E08E0B', '#ff0000'],
+                data: [0,0,0,0]
+            }
+        ]
+    }
+});
+
+var servers = Pterodactyl.servers;
+var nodes = Pterodactyl.nodes;
+
+for (let i = 0; i < servers.length; i++) {
+    setTimeout(getStatus, 200 * i);
```

Comment:
Also I don't think it's even necessary to use setTimeout for this occasion because Javascript executes code sequentially. If you absolutely need them to execute in order, then make the ajax calls synchronous.

Reply:
The requests get sent too fast without a delay, resulting in 404 or 403 errors from the daemon